website/docs/.gitignore

```diff
@@ -1,22 +1,15 @@
-# Sapper + copied node modules
 .DS_Store
-/node_modules/
-/src/node_modules/@sapper/
-/cypress/screenshots/
-/__sapper__/
-
-# Generated and copied
-extendedConfiguration.bundle.js
-luigi-core
-luigi-client
-docsearch.min.css
-public
-static/assets
-static/versions.json
-*coreStyles*
-navigation-generated.json
-sitemap.xml
-
-# Debug
-yarn-error.log
-debug*.log
+node_modules
+/build
+/.svelte-kit
+/package
+/static
+!/static/global.css
+!/static/favicon-sap.ico
+!/static/images
```

Comment:
Exclamation mark **(!)** seems to suggest that **global.css**, **favicon-sap.ico** and folder **images**, supposed to be under **static** folder committed in this PR. I cannot see these files here. seem deleted. Is this intended?

And why not go for simply the following instead of 4 **(!)** for static folder: 
`/static/luigi-client/`

Reply:
also **static/images** folder seems deleted. Is it needed somehow ?

Reply:
global.css, favicon-sap.ico and folder images are in this commit but there wasn't any changes (if you compare it to master branch) so you don't see it here. But if you check it inside folders there are all these files and folders which was inside static folder before all changes.

Reply:
true. then ignore this.